gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleWrapperPatcher.java

```diff
@@ -53,14 +60,17 @@ public abstract class GradleWrapperPatcher {
     interface Params {
 
         @InputFile
-        RegularFileProperty getOriginalGradlewScript();
+        Property<File> getOriginalGradlewScript();
```

Comment:
Why `RegularFileProperty` -> `Property<File>`?

Reply:
I switched to using the outputs from the WrapperTask (instead of hardcoding the paths): https://github.com/palantir/gradle-jdks/pull/360/files#diff-d14af7703cef8b606999ab3937204e21f209c24a65a6196c2c63b24ac0fd8c40R111-R112 which are of `Property<File>` instead of `RegularFileProperty`

Reply:
You can use `.fileProvider(` rather than `.set(`.